cfgmgr/vlanmgr.cpp

```diff
@@ -669,6 +674,307 @@ void VlanMgr::doVlanMemberTask(Consumer &consumer)
     }
 }
 
+void VlanMgr::doVlanPacPortTask(Consumer &consumer)
+{
+    SWSS_LOG_ENTER();
+
+    auto it = consumer.m_toSync.begin();
+    while (it != consumer.m_toSync.end())
+    {
+        auto &t = it->second;
+        string alias = kfvKey(t);
+        string op = kfvOp(t);
+
+        SWSS_LOG_DEBUG("processing %s operation %s", alias.c_str(),
+                                   op.empty() ? "none" : op.c_str());
+
+        if (op == SET_COMMAND)
+        {
+            string learn_mode;
+            for (auto i : kfvFieldsValues(t))
+            {
+                if (fvField(i) == "learn_mode")
+                {
+                    learn_mode = fvValue(i);
+                }
+            }
+            if (!learn_mode.empty())
+            {
+                SWSS_LOG_NOTICE("set port learn mode port %s learn_mode %s\n", alias.c_str(), learn_mode.c_str());
+                vector<FieldValueTuple> fvVector;
+                FieldValueTuple portLearnMode("learn_mode", learn_mode);
+                fvVector.push_back(portLearnMode);
+                m_appPortTableProducer.set(alias, fvVector);
```

Comment:
What about the FDB learnt on the port, prior to PAC being enabled on it? Do we ensure those get flushed when PAC is enabled and port is set to s/w learning mode

Reply:
It should be done in the orchagent code while processing learn mode request, I will add that as well.

Reply:
@praveenraja1 , the change related to flushing FDB on learning mode set is a common fix needed in OA, so its not really a part of PAC implementation. I will create a separate PR for that.